mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
```

Comment:
The error message should be consistent throughout the codebase. In the `removeOrder` function, the error message is `"Order not found"`, but in the `orderById` function, the error message is `errorHandler(err)`. It would be better to use `errorHandler("Order not found")` for consistency.